<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1572</width>
    <height>1109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QWidget" name="mainView" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTabWidget" name="tab_widget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="elideMode">
          <enum>Qt::ElideNone</enum>
         </property>
         <widget class="QWidget" name="display">
          <attribute name="title">
           <string>Display</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="OSGWidget" name="display_widget" native="true">
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>650</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="measurements_table">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>400</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="cursor" stdset="0">
              <cursorShape>ArrowCursor</cursorShape>
             </property>
             <property name="contextMenuPolicy">
              <enum>Qt::DefaultContextMenu</enum>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustToContents</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="dragEnabled">
              <bool>false</bool>
             </property>
             <property name="dragDropMode">
              <enum>QAbstractItemView::NoDragDrop</enum>
             </property>
             <property name="defaultDropAction">
              <enum>Qt::IgnoreAction</enum>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="verticalScrollMode">
              <enum>QAbstractItemView::ScrollPerItem</enum>
             </property>
             <property name="gridStyle">
              <enum>Qt::SolidLine</enum>
             </property>
             <column>
              <property name="text">
               <string>Visibility</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Type</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Category</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Temperature</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Result</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Comments</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="analysis">
          <attribute name="title">
           <string>Analysis</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTextEdit" name="measures_edit">
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1572</width>
     <height>33</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuEfef">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="open_3d_model_action"/>
    <addaction name="open_measure_file_action"/>
    <addaction name="open_project_action"/>
    <addaction name="open_recent_file_action"/>
    <addaction name="save_measure_file_action"/>
    <addaction name="save_project_action"/>
    <addaction name="close_3d_model_action"/>
    <addaction name="quit_action"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="data_export_action"/>
   </widget>
   <widget class="QMenu" name="menuLanguages">
    <property name="title">
     <string>Languages</string>
    </property>
    <addaction name="french_action"/>
    <addaction name="english_action"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="user_manual_action"/>
    <addaction name="about_action"/>
   </widget>
   <widget class="QMenu" name="menu3DMETRICS">
    <property name="title">
     <string>3DMETRICS</string>
    </property>
   </widget>
   <addaction name="menu3DMETRICS"/>
   <addaction name="menuEfef"/>
   <addaction name="menuData"/>
   <addaction name="menuLanguages"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="draw_segment_action"/>
   <addaction name="draw_area_action"/>
   <addaction name="draw_interest_point_action"/>
   <addaction name="cut_area_action"/>
   <addaction name="zoom_in_action"/>
   <addaction name="zoom_out_action"/>
   <addaction name="resize_action"/>
   <addaction name="crop_action"/>
   <addaction name="cancel_button"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="open_3d_model_action">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon_open.png</normaloff>:/icons/ressources/icon_open.png</iconset>
   </property>
   <property name="text">
    <string>Open a 3d Model</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="open_recent_file_action">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon_recentopen.ico</normaloff>:/icons/ressources/icon_recentopen.ico</iconset>
   </property>
   <property name="text">
    <string>Open Recent file</string>
   </property>
  </action>
  <action name="save_measure_file_action">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon_save.png</normaloff>:/icons/ressources/icon_save.png</iconset>
   </property>
   <property name="text">
    <string>Save Measure file </string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="close_3d_model_action">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon_close.jpg</normaloff>:/icons/ressources/icon_close.jpg</iconset>
   </property>
   <property name="text">
    <string>Close 3d Model</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="quit_action">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon_quit.png</normaloff>:/icons/ressources/icon_quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="data_export_action">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon_export.png</normaloff>:/icons/ressources/icon_export.png</iconset>
   </property>
   <property name="text">
    <string>Export (.csv or .txt) format</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="french_action">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon_france.png</normaloff>:/icons/ressources/icon_france.png</iconset>
   </property>
   <property name="text">
    <string>French</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="english_action">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon_anglais.ico</normaloff>:/icons/ressources/icon_anglais.ico</iconset>
   </property>
   <property name="text">
    <string>English</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="user_manual_action">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon_usermanual.jpg</normaloff>:/icons/ressources/icon_usermanual.jpg</iconset>
   </property>
   <property name="text">
    <string>User Manual</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="draw_segment_action">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon_segment.png</normaloff>:/icons/ressources/icon_segment.png</iconset>
   </property>
   <property name="text">
    <string>tracer un segment</string>
   </property>
   <property name="toolTip">
    <string>line tool</string>
   </property>
  </action>
  <action name="draw_area_action">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon_polygon.png</normaloff>:/icons/ressources/icon_polygon.png</iconset>
   </property>
   <property name="text">
    <string>tracer une zone</string>
   </property>
   <property name="toolTip">
    <string>surface tool</string>
   </property>
  </action>
  <action name="draw_interest_point_action">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon_ptinteret.png</normaloff>:/icons/ressources/icon_ptinteret.png</iconset>
   </property>
   <property name="text">
    <string>creer un point d'interet</string>
   </property>
   <property name="toolTip">
    <string>interest point tool</string>
   </property>
  </action>
  <action name="cut_area_action">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon_ciseaux.jpg</normaloff>:/icons/ressources/icon_ciseaux.jpg</iconset>
   </property>
   <property name="text">
    <string>decoupage zone</string>
   </property>
   <property name="toolTip">
    <string>zoning waiver</string>
   </property>
  </action>
  <action name="about_action">
   <property name="text">
    <string>About 3DMETRICS</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="open_measure_file_action">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon_file.png</normaloff>:/icons/ressources/icon_file.png</iconset>
   </property>
   <property name="text">
    <string>Open a Measure file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="open_project_action">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon project.png</normaloff>:/icons/ressources/icon project.png</iconset>
   </property>
   <property name="text">
    <string>Open a Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="zoom_in_action">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon loupe +.png</normaloff>:/icons/ressources/icon loupe +.png</iconset>
   </property>
   <property name="text">
    <string>zoom</string>
   </property>
   <property name="toolTip">
    <string>zoom in</string>
   </property>
  </action>
  <action name="zoom_out_action">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon_loupe -.jpg</normaloff>:/icons/ressources/icon_loupe -.jpg</iconset>
   </property>
   <property name="text">
    <string>dezoom</string>
   </property>
   <property name="toolTip">
    <string>zoom out</string>
   </property>
  </action>
  <action name="resize_action">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon_redimensionnement.jpg</normaloff>:/icons/ressources/icon_redimensionnement.jpg</iconset>
   </property>
   <property name="text">
    <string>redimensionnement</string>
   </property>
   <property name="toolTip">
    <string>resize</string>
   </property>
  </action>
  <action name="crop_action">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon.jpeg</normaloff>:/icons/ressources/icon.jpeg</iconset>
   </property>
   <property name="text">
    <string>recadrage</string>
   </property>
   <property name="toolTip">
    <string>reframe</string>
   </property>
  </action>
  <action name="save_project_action">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon_enregistrer_projet.png</normaloff>:/icons/ressources/icon_enregistrer_projet.png</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen_Recent_Project">
   <property name="text">
    <string>Open Recent Project</string>
   </property>
  </action>
  <action name="cancel_button">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icons/ressources/icon_close.jpg</normaloff>:/icons/ressources/icon_close.jpg</iconset>
   </property>
   <property name="text">
    <string>cancel active tool</string>
   </property>
   <property name="toolTip">
    <string>Cancel currently active tool</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OSGWidget</class>
   <extends>QWidget</extends>
   <header>OSGWidget/OSGWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
